tck/src/main/x/tck/union/Basic.x

```diff
@@ -0,0 +1,21 @@
+/**
+ * Very basic union tests.
+ */
+class Basic {
+
+    void run() {
+        basic();
+    }
+
+    // -----------------------------
+    void basic() {
+        typedef (String | Int) as StrInt;
+        StrInt id( StrInt x ) { return x; }
+        StrInt x = id(7);
+        assert x==7;
```

Comment:
Again, completely trivial and tiny style issue (for spaces): the rest of the tck would look like that:

    assert x == 7;
    assert x != id("abc");
    assert x == "def";

Reply:
changed